h2h.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
+import swifter
+
+class HeadToHead():
+    def __init__(self):
+        pass
+
+    """
+    Creates the head to head features for each fighter in the dataset
+
+    Usage:
+        df = HeadToHead().create_h2h_feats(df)
+    """
+    def create_h2h_feats(self, df):
+        target_df = df
+
```

Comment:
do we want ot change this to the copy method?

Reply:
other than that lgtm

Reply:
Ok bet I'll do that after I finish the test